setup.py

```diff
@@ -40,6 +40,8 @@
             "isort>=5.13.2",
             "pre-commit>=3.7.1",
             "torch-tb-profiler>=0.4.1",
+            "pytest>=7.1.2",
```

Comment:
awesome! we can also remove `pre-commit` and `torch-tb-profiler`. I forgot to do so. Otherwise LGTM!